src/settings.ts

```diff
@@ -87,6 +87,8 @@ export const DUST_STORM_TEXT_1 = `A dust storm is ripping across the ${PLAYA_VEN
 export const DUST_STORM_TEXT_2 =
   "Your only option is to seek shelter in a nearby venue!";
 
+export const GITHUB_MAIN_STAGE_NAME = "Main Stage";
```

Comment:
if this PR is for `staging`, than the logic related to `GITHUB_MAIN_STAGE_NAME` should be removed as it's a GitHub specific request.
if this PR is for `env/github`, than the base branch should be changed.

Reply:
created another PR for env/github specifically https://github.com/sparkletown/sparkle/pull/1741
thx